private-templates-service/client/src/components/NavBar/NavBar.tsx

```diff
@@ -97,10 +99,10 @@ const NavBar = (props: NavBarProps) => {
         <Banner>
           <MobileBanner>
             <StyledLogo src={Logo} />
-            <Styledh1>{props.template ? props.template.name : props.currentPageTitle}</Styledh1>
-            <Styledh2>{props.version ? "Version " + props.version : ""}</Styledh2>
+            <Styledh1>{(props.templateID === "" && "Untitled Card") || props.templateName}</Styledh1>
```

Comment:
Move string "Untitled Card" to string assets file?

Reply:
Done, Also moved all strings from SaveModal.tsx to the string assets file, nice catch